app/src/main/res/layout/mpower2_add_note.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent" android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/add_note_label"
+        style="@style/Widget.MPower.TextView.SelectionSectionTitle"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="28dp"
```

Comment:
For some of the fragments this doesn't work. For example on a Selection Step when a button is pressed it turns light purple and the entire width of the screen should have it's color changed.

Reply:
Padding is defined within the ui element (margin outside of it), so the color would apply to the full width

Reply:
Also I don't know if I can set the margins of 28dp on the parent since the back arrow button only has a 20dp margin and I think setting the padding would interfere with that.